pkg/controller/operators/olm/operator.go

```diff
@@ -370,6 +371,14 @@ func (a *Operator) GetReplaceFinder() csvutility.ReplaceFinder {
 	return a.csvReplaceFinder
 }
 
+func (a *Operator) RegisterCSVWatchNotification(csvNotification csvutility.WatchNotification) {
```

Comment:
For now this supports a single client, in the future we can extend it to support multiple clients.